src/Application/Clans/Commands/Armory/ReturnUnusedClanArmoryItemsCommand.cs

```diff
@@ -0,0 +1,46 @@
+using Crpg.Application.Common.Interfaces;
+using Crpg.Application.Common.Mediator;
+using Crpg.Application.Common.Results;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using LoggerFactory = Crpg.Logging.LoggerFactory;
+
+namespace Crpg.Application.Clans.Commands.Armory;
+
+public record ReturnUnusedClanArmoryItemsCommand : IMediatorRequest
+{
+    public TimeSpan Timeout { get; init; }
+
+    internal class Handler : IMediatorRequestHandler<ReturnUnusedClanArmoryItemsCommand>
+    {
+        private static readonly ILogger Logger = LoggerFactory.CreateLogger<ReturnUnusedClanArmoryItemsCommand>();
+
+        private readonly ICrpgDbContext _db;
+
+        public Handler(ICrpgDbContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<Result> Handle(ReturnUnusedClanArmoryItemsCommand req, CancellationToken cancellationToken)
+        {
+            var now = DateTime.UtcNow;
+            var users = await _db.Users
+                .Include(u => u.ClanMembership!).ThenInclude(cm => cm.ArmoryBorrowedItems).ThenInclude(bi => bi.UserItem!).ThenInclude(ui => ui.EquippedItems)
+                .Where(bi => bi.ClanMembership!.ArmoryBorrowedItems.Count > 0 && (now - bi.UpdatedAt) > req.Timeout)
+                .ToArrayAsync(cancellationToken);
```

Comment:
AsSplitQuery is still required as these chain include will create an sql request with an insane amount of duplicated data that will hurt massively performance